src/main/webapp/aboutus.html

```diff
@@ -35,11 +35,11 @@ <h2>Teammate A Name</h2>
       <li>Aspirational Hobby: </li>
       <li>Ask me About: </li>
     </ul>
-    <h2>Teammate B Name</h2>
+    <h2>Brandy Huang</h2>
     <ul>
-      <li>Summer Feelz: </li>
-      <li>Aspirational Hobby: </li>
-      <li>Ask me About: </li>
+      <li>Summer Feelz: Exams、Rainy、"The Secret Life of Pets" </li>
+      <li>Aspirational Hobby: I think I'll learn how to fiy a plane if I have unlimited time and money. Being a pilot is a dream in my childhood and it sounds really cool to flying in the endless sky.  </li>
```

Comment:
Typo: "fiy" should be "fly".
```suggestion
      <li>Aspirational Hobby: I think I'll learn how to fly a plane if I have unlimited time and money. Being a pilot is a dream in my childhood and it sounds really cool to flying in the endless sky.  </li>
```